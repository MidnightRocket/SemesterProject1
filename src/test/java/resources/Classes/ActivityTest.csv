test1,testDisplay1,itemId,false,2,0
test2,testDisplay2,itemId,true,-2147483648,-5
test3,testDisplay3,itemId,true,-500,9
test4,testDisplay4,itemId,false,234,-3
test5,testDisplay5,itemId,true,0,-3