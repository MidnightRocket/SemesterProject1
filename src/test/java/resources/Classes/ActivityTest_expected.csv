test1,testDisplay1,false,itemId,2,0
test2,testDisplay2,true,itemId,successPoints cannot be negative,powerCost cannot be negative
test3,testDisplay3,true,itemId,successPoints cannot be negative,9
test4,testDisplay4,false,itemId,234,powerCost cannot be negative
test5,testDisplay5,true,itemId,0,powerCost cannot be negative